### PR TITLE
Added unique urls for recipes and users

### PR DESCRIPTION
**Pull request by k12kekeal**
Still working out how to handle the links on the user info toolbar, but that should be easier once we have the store set up!

Also I imported the search bar into the recipe page just for layout's sake, but that can be changed too!

### CODE REVIEW

File: front-end/src/components/CenterConsole/SingleRecipe.js [hunk 1]
```diff
@@ -4,11 +4,11 @@ import StarRateIcon from '@material-ui/icons/StarRate';
 import { Link } from 'react-router-dom';
 
 const SingleRecipe = ({oneRecipe}) => {
-
+  const link = `/recipe/${oneRecipe.recipeId}`;
 
   return (
     <Grid item xs={4}>
-      <Link to='/recipe'>
+      <Link to={link}>
```
Comment (chaomonica) [inline front-end/src/components/CenterConsole/SingleRecipe.js:11]: Awesome idea!

File: front-end/src/components/UserInfoToolbar/UserInfoToolbar.jsx [hunk 2]
```diff
@@ -31,24 +32,40 @@ export default function LeftToolbarLanding() {
...
-          <Button onClick={()=>alert('popup modal with picture upload')}>
-                add picture
+          <Button onClick={(e) => {
+            e.preventDefault();
+            toggleVisibility(!isVisible);
+          }}>
+            change avatar
           </Button>
+          {isVisible ?
+            <div>
```
Comment (chaomonica) [inline front-end/src/components/UserInfoToolbar/UserInfoToolbar.jsx:58]: very cool!

File: front-end/src/components/Recipe/RecipeData.jsx [hunk 3]
```diff
@@ -1,13 +1,26 @@
 import {recipe} from '../../data/recipeDummyData.js';
 import React from 'react';
+import { Link, useParams } from 'react-router-dom';
 
 const RecipeData = () => {
-
+  // var recipe;
```
Comment (chaomonica) [inline front-end/src/components/Recipe/RecipeData.jsx]: are these still needed?

**Review by chaomonica — APPROVED** [review]

File: front-end/src/components/Recipe/RecipeData.jsx (hunk 3 above)
Comment (k12kekeal) [reply front-end/src/components/Recipe/RecipeData.jsx]: Oops, leftovers from trying to switch to userCookbook!

**Pull request merged**